File: UNIT 2/C/kalseder.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int kode;
    
    printf("Masukan kode hari ini (1,2,...,7): ");
    scanf("%d", &kode);

    switch (kode)
    {
    case 1:
        printf("\nIni kode hari SENIN");
        break;
    case 2:
        printf("\nIni kode hari SELASA");
        break;
    case 3:
        printf("\nIni kode hari RABU");
        break;
    case 4:
        printf("\nIni kode hari KAMIS");
        break;
    case 5:
        printf("\nIni kode hari JUMAT");
        break;
    case 6:
        printf("\nIni kode hari SABTU");
        break;
    case 7:
        printf("\nIni kode hari MINGGU");
        break;
    default:
        printf("\nKode yang Anda masukkan tidak dikenali\n");
    }
    
    return 0;
}

File: UNIT 2/py/kalseder.py
def hari (i):
    switcher = {
        1: "Ini kode hari SENIN",
        2: "Ini kode hari SELASA",
        3: "Ini kode hari RABU",
        4: "Ini kode hari KAMIS",
        5: "Ini kode hari JUMAT",
        6: "Ini kode hari SABTU",
        7: "Ini kode hari AHAD"
    }
    return switcher.get(i, "Kode yang anda masukkan tidak dikenal")
print(hari(7))
